runtime/src/main/java/org/corfudb/runtime/view/stream/AddressMapStreamView.java

```diff
@@ -381,8 +381,8 @@ private boolean isTrimCoveredByCheckpoint(long trimMark) {
      * @return whether this stream is capable of being checkpointed
      */
     private boolean isCheckpointCapable() {
-        return !getId().equals(ObjectsView.TRANSACTION_STREAM_ID)
-                && getStreamOptions().isCheckpointCapable();
+        // Subscription tag streams are not checkpoint(able) (used for streaming) this is set on the StreamSubscription
+        return getStreamOptions().isCheckpointCapable();
```

Comment:
isCheckpointCapable of what? it would be great to rename to something better

Reply:
if the stream is checkpoint capable. Renaming this would break NSX code as this is used by the DCN layer. Once DCNs are removed we can rename as this will only be used within Corfu code.